Environment/Environment.pillar

```diff
@@ -442,43 +443,42 @@ your image first!
 !!!The Inspector
 
 (do this) As an illustration of what you can do with an inspector, type
-==TimeStamp now== in a workspace, and then right-click and choose ==Inspect
-it==.
+==DateAndTime now== in a playground, and then right-click and choose ==Inspect
+it== or ==Do it and go==.
```

Comment:
I'll open a new issue about it. It's a feature of the playground, that open an inspector inside the playground, so you can explore your code and still have it in the same windows. 
Good to use when you've too many windows.